packages/client3/components/design-system/snackbar/snackbar.tsx

```diff
@@ -0,0 +1,21 @@
+import clsx from "clsx";
+import React from "react";
+import { ToastContainer } from "react-toastify";
+
+import { ISnackbar } from "./types";
+
+export function Snackbar(props: ISnackbar) {
```

Comment:
Would recommend putting `ISnackbar ` definition within this component itself as it's specific to snackbar.